www/rounds.php

```diff
@@ -3,11 +3,24 @@
 require_once '../lib/Util.php';
 
 $rounds = Model::factory('Round')
-        ->order_by_desc('start')
+        ->order_by_desc('duration')
         ->find_many();
 
 $user = Session::getUser();
 
+
+const STATUS_CLASSES = [
+  Round::STATUS_EXPIRED => 'bg-danger',
+  Round::STATUS_ONGOING => 'bg-success',
+  Round::STATUS_UPCOMING => 'bg-info'
+];
+
+function getRoundStateClass($round){
+  return STATUS_CLASSES[$round->getStatus()];
+}
+
+
+SmartyWrap::registerPlugin('modifier', 'round_class', 'getRoundStateClass');
```

Comment:
Perhaps this would look better as a method on `Round`, i.e. `Round->getStatusClass()`? Then it would be reusable wherever rounds are displayed (e.g. if we ever decide to display all the rounds that a problem is part of). And we wouldn't need the Smarty plugin.

No strong preference, though.